applications/google_rise_2016.md

```diff
@@ -0,0 +1,459 @@
+international, we hope to eventually expand our reach to the millions.
+
+**Projected demographics of the youth reached by this project**
+
+_Please use approximate percentages to list the demographics of the youth that
+your proposed project will serve this year, such as age ranges, race/ethnicity,
+gender etc. For example, 25% boys, 75% girls._
+
+We project our demographics in 2016 to mirror those in 2015, though we do plan
+on serving more young women. Our demographics in 2015 are as follows:
+
+- 32% women (compare with 20% female AP CS test takers)
+- 37% black/Hispanic (compare with 13% black/Hispanic AP CS test takers in USA)
+
+**What are the anticipated results (objectives/outputs) for this project? (list
+up to 5)**
+
+Our goal is to turn members of Hack Clubs into hackers: people who create what
+they see missing in the world and aren't afraid to break the rules. We'll know
+this is happening if:
+
+- Members are continuing to write code outside of Hack Club
+- Members are working on long-term projects and publishing them
+- Members are winning hackathons and other competitions
+- Members are hosting their own events (hackathons, meetups, etc)
+
+**How do you plan to measure and evaluate your results?**
+
+We'll continue to measure our primary KPIs: the number of clubs and the stories
+we're hearing from club leaders and members.
+
+Additionally, we'll begin measuring attendance of Hack Clubs week-to-week and
+their attrition rates.
+
+### Sustainability and Scalability
+
+Help us understand how the project you are proposing fits into the larger
+context of your organization.
+
+**What is the purpose of your proposal?**
+
+- [ ] Create or add a new program to the organization
+- [X] Expand an existing program to more students, a new population, or a new
+  geography
+- [ ] Secure funding for an existing program
+- [ ] Other
+
+**How do you plan to sustain your project beyond 2016 and beyond this grant?**
+
+Grants and company sponsorships. We plan to continue to keep Hack Clubs free for
+students by not collecting membership dues.
+
+We're thinking about building a regional grant pipeline: a data-driven process
+for consistently applying and winning grants in areas we have clubs. We're still
+in the ideation phase for this.
+
+Other ideas include creating a Hack Club gap year, where club members and
+leaders could apply to take a gap year before college and work on Hack Club.
+Fellowships would be sponsored by companies and foundations. We've received
+preliminary interest from companies interested in sponsoring this sort of
+program.
+
+**Does this project have a post-engagement strategy to keep students connected
+with computer science? If yes, please describe.**
+
+For many programs, we are the post-engagement strategy. Our clubs meet weekly or
+bi-weekly throughout the school year and we've received partnership interest
+from a variety of computer science summer programs wanting their students to
```

Comment:
Is it helpful to include which programs? Or is that jumpin the gun cuz the partnerships are not concrete yet?